gluster_exporter/metric_brick.go

```diff
@@ -167,6 +190,24 @@ func BrickUtilization() {
 					glusterBrickInodesTotal.With(lbls).Set(usage.InodesAll)
 					glusterBrickInodesFree.With(lbls).Set(usage.InodesFree)
 					glusterBrickInodesUsed.With(lbls).Set(usage.InodesUsed)
+					if usage.Used >= maxBrickUsed {
+						maxBrickUsed = usage.Used
+					}
+				}
+			}
+			if subvol.Type == "Arbiter" {
```

Comment:
subvolume type for Arbiter volume is also Replicate. Skip only exporting metrics from Arbiter brick(Other two data bricks will export the data in that sub volume). See above comment

Reply:
Ack